catalog.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="css/catalog.css">
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="icon" href="/img/icons/half-logo-icon.svg" type="image/svg+xml">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Poppins:ital,wght@0,400;0,500;0,600;0,900;1,900&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+</head>
+<body class="body">
+  <header class="header">
+    <a href="https://binabox.com" class="binabox">
+      <svg class="logo" width="188" height="42">
+          <use href="/img/icons/sprite.svg#logos"></use>
+      </svg>  
+    </a>
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="navigation-item"><a href="blog-article.html" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="catalog.html" class="nav-link card--active">Catalog</a></li>
+        <li class="navigation-item"><a href="contact-us.html" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <ul class="header-buttons">
+      <li class="search-form"><form action="" class="search-button" method="get">
+        <input type="search" placeholder="Search" class="button button-search">
+        </form>
+      </li>
+      <li class="discord-button">
+        <a href="https://discord.com" target="_blank" class="discord">Discord</a>
+      </li>
+      <li class="connect-button">
+        <button class="connect" type="button">Connect</button>
+      </li>
+    </ul>
+  </header>
+  <main class="content">
+    <div class="banner">
+        <h2 class="banner__header">Explore catalog</h2>
+        <ul class="banner__navigation">
+            <li class="banner__nav-item"><a href="index.html" class="nav-link">Home</a></li>
+            <li class="banner__nav-item"><a href="" class="nav-link card--nonactive">Catalog</a></li>
+        </ul>
+    </div>
+    <section class="catalog__main-section">
+        <form action="" method="post" class="left-block-filters">
+            <div class="clothing">
```

Comment:
@Alinchik1 explore [FIELDSET tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), it's a good option to group inputs by types

Reply:
added